test/user-profile/reducers/profile.unit.spec.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+
+import reducer from '../../../src/js/user-profile/reducers/profile.js';
+import { UPDATE_LOGGEDIN_STATUS } from '../../../src/js/login/actions';
+import { PROFILE_LOADING_FINISHED } from '../../../src/js/user-profile/actions';
+
+describe('Profile reducer', () => {
```

Comment:
I think we know we want to refactor that code anyway, so I think it's ok to wait.